packages/os.go

```diff
@@ -16,10 +26,12 @@ func init() {
 		"Create":            os.Create,
 		"DevNull":           os.DevNull,
 		"Environ":           os.Environ,
+		"ErrClosed":         os.ErrClosed,
 		"ErrExist":          os.ErrExist,
 		"ErrInvalid":        os.ErrInvalid,
 		"ErrNotExist":       os.ErrNotExist,
 		"ErrPermission":     os.ErrPermission,
+		"Executable":        os.Executable,
```

Comment:
Some of the added function should be added to `// +build !appengine` like Executable, Unsetenv, and etc.

Reply:
Ok. Then just ask, how to know the list of `// +build !appengine`